conpot/tests/test_taxii.py

```diff
@@ -40,9 +40,9 @@ def test_stix_transform(self):
         config.read(config_file)
         config.set('taxii', 'enabled', True)
         config.set('taxii', 'use_contact_info', True)
```

Comment:
hm, because this is test for stix, I would set use_contact_info, and contact_mail in stix

but it looks they are not needed/used anyway @glaslos ? @johnnykv ?